src/angularComponent/ngcOmniboxController.js

```diff
@@ -0,0 +1,11 @@
+import OmniboxController from 'src/coreComponent/OmniboxController.js';
```

Comment:
In fact, I would more prefer that you proceed to write things **without** thinking about the "core" stuff first, because I don't think either of us can fully predict what makes sense as core and what does not yet. After we have working code in place, we can then do an architecture review to figure out exactly what we can extract into the framework-agnostic core.